roles/cpu/tasks/cpu.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Execute CPU healthcheck command on Cisco IOS XR device
+  cisco.iosxr.iosxr_command:
+    commands:
+      - "{{ cpu_healthcheck_command }}"
```

Comment:
we need to have these commands called dynamically as these could have slight differences based on OS.